src/config/config.ts

```diff
@@ -47,6 +47,12 @@ export const appConfig = {
       "https://digital-research-books-beta.nypl.org/about?source=catalog",
     circulatingCatalog: "https://nypl.na2.iiivega.com/",
     legacyCatalog: "https://legacycatalog.nypl.org/",
+    loginUrl:
+      process.env.LOGIN_BASE_URL + "/login" ||
+      "https://login.nypl.org/auth/login",
```

Comment:
Hard-code the defaults to the QA links for login and logout urls.

Reply:
I take this back.

I want to say to try the pattern that's used for other env vars instead where its
```
loginUrl: {
  development: ...
  production: ..
}
```

but this is complicated by the fact that it uses an env var to generate the string. This probably needs a larger discussion with other devs about what the defaults should be.